lib/king_konf/config_file_loader.rb

```diff
@@ -11,7 +11,11 @@ def load_file(path, environment = nil)
       # First, load the ERB template from disk.
       template = ERB.new(File.new(path).read)
 
-      data = YAML.load(template.result(binding))
+      begin
+        data = YAML.load(template.result(binding), aliases: true)
+      rescue ArgumentError
+        data = YAML.load(template.result(binding))
+      end
```

Comment:
Can you add a comment in the code explaining why this is necessary?

Reply:
Sure, will do it later on.

Reply:
Sorry, where's the changelog?